From: SeongJae Park <sj@kernel.org>
Date: Tue, 15 Jul 2025 10:17:13 -0700
Subject: [PATCH] ==== damon_callback removal fixups ====

mm/damon: kernel-doc comments fixup for damon_callback removal

Patch series "mm/damon: remove damon_callback"[1] causes two kernel-doc
comments build warnings[2].  Fix those.

[1] https://lore.kernel.org/20250712195016.151108-1-sj@kernel.org
[2] https://lore.kernel.rog/20250715185549.264260b8@canb.auug.org.au

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_9hj3l5qk | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_9hj3l5qk

diff --git a/hkml_cv_bogus/hkml_cv_bogus_9hj3l5qk b/hkml_cv_bogus/hkml_cv_bogus_9hj3l5qk
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

